cnf/ext/central.mvn

```diff
@@ -54,8 +54,14 @@ org.apache.geronimo.specs:geronimo-jcdi_2.0_spec:1.1
 
 org.apache.ant:ant:1.7.1
 
+org.junit.platform:junit-platform-launcher:1.4.0
```

Comment:
Version 1.5.0 is now available. Should we not be using the latest version?

Reply:
Version 1.4.0 was the lowest version that provided `LegacyXmlReportGeneratingListener`. I think ideally, all of the available JUnit jars should be available as bundles too and the end user can choose which version they want to install and use. But there is probably little harm in bumping this to 1.5.0 in the meantime.

Reply:
As I mentioned above, perhaps we may just want to conditionalpackage that class into our jar and avoid the whole repackaging of a junit jar thing.